packages/forma-36-react-components/src/components/Notification/index.tsx

```diff
@@ -7,42 +9,43 @@ import NotificationManager, {
   SetPositionAction,
   SetDurationAction,
   Intent,
+  Notification,
 } from './NotificationsManager';
 
 interface NotificationsAPI {
-  success: ShowAction;
-  error: ShowAction;
-  close: CloseAction;
-  closeAll: CloseAllAction;
-  setPosition: SetPositionAction;
-  setDuration: SetDurationAction;
+  success: ShowAction<Notification>;
+  error: ShowAction<Notification>;
+  show: ShowAction<Notification>;
+  close: CloseAction<void>;
+  closeAll: CloseAllAction<void>;
+  setPosition: SetPositionAction<void>;
+  setDuration: SetDurationAction<void>;
 }
 
 let initiated = false;
-const internalAPI: Partial<
-  NotificationsAPI & {
-    show: ShowAction;
-  }
-> = {};
+const internalAPI: Partial<NotificationsAPI> = {};
 
 function registerAPI(fnName, fn) {
   internalAPI[fnName] = fn;
 }
 
-function createRoot() {
+function createRoot(callback) {
   const container = document.createElement('div');
   document.body.appendChild(container);
 
-  render(<NotificationManager register={registerAPI} />, container);
+  render(<NotificationManager register={registerAPI} />, container, callback);
```

Comment:
@suevalov 
Could this also be wrapped in a Promise, like `afterInit`?

Reply:
This function is not exposed as external API and uses callback internally